doc/source/concurrency_group_api.rst

```diff
@@ -54,6 +54,64 @@ You can define concurrency groups for asyncio actors using the ``concurrency_gro
         a.f4.remote()  # executed in the "compute" group.
         a.f5.remote()  # executed in the default group.
 
+  .. group-tab:: Java
+
+    This defines two concurrency groups, "io" with max_concurrency=2 and
+    "compute" with max_concurrency=4.  The methods ``f1`` and ``f2`` are
+    placed in the "io" group, and the methods ``f3`` and ``f4`` are placed
+    into the "compute" group. Note that there is always a default
+    concurrency group, which has a default concurrency of 1000 in Python and
+    1 in Java.
```

Comment:
Can we make this paragraph shared? And I'm not sure what `max_concurrency` means because it's not demonstrated in the code snippets. Maybe it's better to change `max_concurrency` to `max concurrency`.